build_script.py

```diff
@@ -26,6 +26,14 @@ def run(command):
     subprocess.check_call(command, shell=True)
 
 
+def _mkdirp(path):
```

Comment:
I do think preceding with an underscore is typically done for "internal" functions, but I may be wrong. `note` was tangential to this change so I left it alone. @practicalswift, what do you think?